content/articles/data-viz-chartify/index.md

```diff
@@ -0,0 +1,336 @@
+
+This dataset was also used in this tutorial piece on [Seaborn](https://www.section.io/engineering-education/seaborn-tutorial/). We would reproduce some of the graphs from the article and add other charts.
+
+Now let us read and display the dataset by running the code below.
+
+```python
+pokemon_df = pd.read_csv('pokemon.csv')
+pokemon_df.head(5)
+```
+We will check our column using the `.columns` and add an `s/n` column using the code snippet below.
+
+```python
+pokemon_df.insert(loc =0, column = 's/n',value = np.arange(len(pokemon_df)))
+```
+### Problem Encounter while using Chartify
+While working with Charify I discovered a problem. I was unable to display my chart. 
+
+when I ran my codes for the first time, I got an error message. If we take a look at the error message, we will see it was a compatibility issue. The latest version of ChromeDriver available only supports Chrome version 93 and my current Chrome browser version is 96.0.4664.45. 
+
+![Error message](/engineering-education/data-viz-chartify/errormessage.png)
+*Screenshot by author*
+
+Why was this affecting chartify�s ability to display the visual? This is because like I mentioned earlier, ChromeDriver is needed for our PNG output (Visual). To resolve this problem, we will need to downgrade our Chrome browser. This can be achieved in 3 easy steps:
+
+- **Step 1**: Uninstall Chrome.
+- **Step 2**: Download Older Chrome Version. You can get this from sites like [FileHippo](https://www.slimjet.com/chrome/google-chrome-old-version.php) and [SlimJet](https://filehippo.com/download_google-chrome/history/) since there is an official repository hosting an older build of the Chrome browser.
+- **Step 3**:  Disable Chrome Auto-Updates. 
+
+You can get a more detailed step-by-step tutorial in this article, [How to Downgrade and Install Older Versions of Chrome?](https://browserhow.com/how-to-downgrade-and-install-older-version-of-chrome/).
+
+### Data Visualization using Chartify
+
+To get started with our charts plotting, type and run the below code snippet.
+
+```python
+ch = chartify.Chart()
+ch.show()
+```
+The code we just ran will give us the output below. 
+
+![Chartify](/engineering-education/data-viz-chartify/chartifyone.png)
+*Screenshot by author*
+
+Chartify makes it relatively easy for first-timers to get started. That's one amazing thing about it. This empty chart that got displayed shows users of the library how they can fill the chart with data. 
+
+Let�s visualize the Pokemon type distribution using a bar chart.
+
+### Pokemon type distribution
+
+A bar chart represents data by using bars of different heights. We would use on to see the distribution of the various pokemon types. First, we need to create a data frame by grouping the Pokemon type 1 and its generation. This can be done using the code snippet below:
```

Comment:
```suggestion
A bar chart represents data by using bars of different heights. We would use on to see the distribution of the various Pokemon types. First, we need to create a data frame by grouping the Pokemon type 1 and its generation. This can be done using the code snippet below:
```